components/speedreader/common/speedreader_panel.mojom

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+module speedreader.mojom;
+
+interface PanelFactory {
+  CreateInterfaces(pending_receiver<PanelHandler> panel_handler);
+};
+
+// Browser-side handler for requests from WebUI page.
+interface PanelHandler {
+  ShowBubble();
+  CloseBubble();
+};
+
+interface SpeedreaderPanel {
+  Foo();
```

Comment:
Use Bar() instead :D

Reply:
not valid anymore. i've added actual APIs in following commits.